src/test/java/com/redhat/fuse/apicurio/jaxrs/tests/GenerateFuseProjectResourceTest.java

```diff
@@ -102,6 +117,14 @@ public void testForRestDSLOutputFromSimpleOpenAPIFile() throws Exception {
 		assertTrue(camelContextXML != null);
 		assertTrue(camelContextXML.trim().length() > 0);
 	}
+	
+	@Test
+	public void testForRemovedHostAndSchemesFromSwagger() throws Exception {
+		Swagger swagger = getCleanedUpSwagger("src/test/resources/todo-swagger.json");
+		assertTrue(swagger != null);
```

Comment:
I think we can use assertNotNull which will provide a slightly better error message